### ISSUE TITLE
Compress jar using pack200

### ISSUE THREAD

**Issue by hrj** [UprootLabs/gngr#120]
This reduces file sizes by a decent amount:
```
pack200 -r -G -O compressed.jar gngr-0.3.8.jar
```

**Comment by vn971** [comment]
Note: this issue could be superseded by https://github.com/UprootLabs/gngr/issues/6

**Comment by hrj** [comment]
Yes, it would be nice if the build process did this automatically.

**Comment by hrj** [comment]
Another option worth evaluating is the `zopfli` compressor. According to [this thread](https://github.com/google/zopfli/issues/29), it should be possible to use [AdvanceComp](https://github.com/amadvance/advancecomp) on jar files.

**Comment by hrj** [comment]
There's a pure java Zopfli compressor too: [CafeUndZopfli](https://github.com/eustas/CafeUndZopfli)
